java/benchmarks/build.gradle

```diff
@@ -9,7 +9,13 @@ repositories {
 }
 
 dependencies {
-    implementation project(':client')
+    def releaseVersion = System.getenv("GLIDE_RELEASE_VERSION");
+
+    if (releaseVersion) {
+        implementation "io.valkey:valkey-glide:" + releaseVersion
```

Comment:
Oops!
benchmark will use `glide-rs` lib compiled locally, not the one we have in artifact
You need to ensure that `applicationDefaultJvmArgs` (line 35) is unset if you're testing a deployment.
You can also skip task depedency on line 30 in such case, but it is not harmful.